apps/wallet-mobile/src/components/CopyButton.tsx

```diff
@@ -29,17 +33,32 @@ export const CopyButton = ({value, onCopy, children, style}: CopyButtonProps) =>
   )
 }
 
-const AnimatedCopyButton = ({
+export const AnimatedCopyButton = ({
   onCopy,
   children,
   style,
   isCopying,
-}: Omit<CopyButtonProps, 'value'> & {isCopying: boolean}) => {
-  const {colors} = useStyles()
+  message,
+}: Omit<CopyButtonProps, 'value'> & {isCopying: boolean; message?: string | null | undefined}) => {
+  const {styles, colors} = useStyles()
 
   return (
     <TouchableOpacity onPress={onCopy} disabled={isCopying} testID="copyButton" style={style}>
-      {isCopying ? <Icon.CopySuccess size={26} color={colors.gray} /> : <Icon.Copy size={26} color={colors.gray} />}
+      {isCopying ? (
+        <View style={styles.rowContainer}>
+          {message != null ? (
```

Comment:
otherwise empty strings i.e '' will end up show an empty tooltip, I'd advise to rename the prop from `message` to `withAnimatedSuccessMessage`, message in this case is too generic.
```suggestion
          {!isEmptyString(message) ? (
```

Reply:
Thanks @stackchain . Will include this along with the other changes.